veileder.html

```diff
@@ -0,0 +1,26 @@
+<!doctype html>
```

Comment:
samme her, jeg ville kanskje kalle den for index-veileder.html

Reply:
Renamet